src/Tizen.NUI.Components/Theme/ThemeDark.cs

```diff
@@ -371,6 +371,11 @@ public Theme Create()
                 },
             });
 
+            theme.AddStyleWithoutClone("Tizen.NUI.Components.ContentPage", new AppBarStyle()
```

Comment:
Please use ViewStyle instead of AppbarStyle.

Reply:
Thank you for your comment.
The code has been fixed.